nucypher/characters/control/specifications/fields/treasuremap.py

```diff
@@ -40,7 +41,7 @@ def _deserialize(self, value, attr, data, **kwargs):
 
     def _validate(self, value):
 
-        splitter = BytestringSplitter(Signature,
+        splitter = BytestringSplitter(signature_splitter,
```

Comment:
@fjarri - I modified this for Porter, interested to get your thoughts there. -see https://github.com/derekpierre/nucypher/blob/porter-impl/nucypher/characters/control/specifications/fields/treasuremap.py#L37